app/src/main/java/io/seqera/wavelit/App.java

```diff
@@ -256,6 +258,12 @@ protected ContainerConfig prepareConfig() {
         if( size>=10 * _1MB )
             throw new RuntimeException("Compressed container layers cannot exceed 10 MiB");
 
+        //validate entrypoint
+        if(entryPoint == null && entryPoint.isEmpty()){
```

Comment:
Actually, empty string is a legit value because it allows overriding the container entrypoint with the default one, see [here](https://stackoverflow.com/a/37636548/395921).

Reply:
ok understood